statsmodels/stats/tests/test_robustros.py

```diff
@@ -0,0 +1,440 @@
+            0.340 False
+            0.457 False
+            0.514 False
+            0.629 False
+            0.638 False
+            0.774 False
+            0.788 False
+            0.900  True
+            0.900  True
+            0.900  True
+            1.000  True
+            1.000  True
+            1.000  True
+            1.000  True
+            1.000  True
+            1.000 False
+            1.000  True
+            1.000  True
+            1.000  True
+            1.000  True
+            1.000  True
+            1.000  True
+            1.000  True
+            1.000  True
+            1.000  True
+            1.000  True
+            1.000  True
+            1.000  True
+            1.100 False
+            2.000 False
+            2.000 False
+            2.404 False
+            2.860 False
+            3.000 False
+            3.000 False
+            3.705 False
+            4.000 False
+            5.000 False
+            5.960 False
+            6.000 False
+            7.214 False
+           16.000 False
+           17.716 False
+           25.000 False
+           51.000 False""")
+        data = pd.read_csv(datastring, sep='\s+')
+        self.obs = data.res
+        self.cen = data.cen
+
+        self.known_cohn_nuncen_above = np.array([9., 0.0, 18., np.nan])
+        self.known_cohn_nobs_below = np.array([3., 15., 32., np.nan])
+        self.known_cohnncen_equal = np.array([3., 3., 17., np.nan])
+        self.known_cohn_prob_exceedance = np.array([0.84, 0.36, 0.36, 0])
+        self.known_plot_pos = np.array([
+            0.036,  0.040,  0.071,  0.080,  0.107,  0.120,  0.142,  0.160,
+            0.178,  0.213,  0.249,  0.284,  0.320,  0.320,  0.356,  0.391,
+            0.427,  0.462,  0.480,  0.498,  0.533,  0.569,  0.604,  0.208,
+            0.256,  0.304,  0.352,  0.400,  0.448,  0.496,  0.544,  0.592,
+            0.659,  0.678,  0.697,  0.716,  0.735,  0.754,  0.773,  0.792,
+            0.811,  0.829,  0.848,  0.867,  0.886,  0.905,  0.924,  0.943,
+            0.962,  0.981
+        ])
+        self.known_modeled = np.array([
+            0.01907990,  0.03826254,  0.06080717,  0.10100000,  0.13600000,
+            0.34000000,  0.45700000,  0.51400000,  0.62900000,  0.63800000,
+            0.77400000,  0.78800000,  0.08745914,  0.25257575,  0.58544205,
+            0.01711153,  0.03373885,  0.05287083,  0.07506079,  0.10081573,
+            1.00000000,  0.13070334,  0.16539309,  0.20569039,  0.25257575,
+            0.30725491,  0.37122555,  0.44636843,  0.53507405,  0.64042242,
+            0.76644378,  0.91850581,  1.10390531,  1.10000000,  2.00000000,
+            2.00000000,  2.40400000,  2.86000000,  3.00000000,  3.00000000,
+            3.70500000,  4.00000000,  5.00000000,  5.96000000,  6.00000000,
+            7.21400000, 16.00000000, 17.71600000, 25.00000000, 51.00000000
+        ])
+        self.known_plot_pos.sort()
+        self.known_modeled.sort()
+
+
+class testROSNoNDs(CheckROSMixin):
```

Comment:
class names are still not pep-8, should start with capital letter, and if it's composed of several nouns, each should start with a capital letter.

can you add a `#` comment with a hint to what the name means? mainly as aid in reading and browsing code.